scripts/variantstore/wdl/extract/gvs_avros_to_vds.py

```diff
@@ -0,0 +1,375 @@
+
+import os
+from typing import List
+import hail as hl
+from hail.genetics.reference_genome import reference_genome_type
+
+from hail.typecheck import typecheck, nullable, oneof, dictof, anytype, sequenceof, enumeration, sized_tupleof, numeric, table_key_type, char
+
+
+@typecheck(refs=sequenceof(sequenceof(str)),
+           vets=sequenceof(sequenceof(str)),
+           sample_mapping=sequenceof(str),
+           site_filtering_data=sequenceof(str),
+           vqsr_filtering_data=sequenceof(str),
+           vqsr_tranche_data=sequenceof(str),
+           final_path=str,
+           tmp_dir=str,
+           truth_sensitivity_snp_threshold=float,
+           truth_sensitivity_indel_threshold=float,
+           reference_genome=reference_genome_type,
+           partitions_per_sample=numeric,
+           intermediate_resume_point=int,
+           skip_final_merge=bool,
+           unphase=bool,
+           )
+
+
+def import_gvs(refs: 'List[List[str]]',
```

Comment:
Weren't we going to rename this something other than import_gvs at this stage?  It's obviously not necessary, but I thought I remember the team discussing that